pkg/anypoint/client.go

```diff
@@ -42,15 +37,14 @@ type Client interface {
 	GetExchangeAsset(groupID, assetID, assetVersion string) (*ExchangeAsset, error)
 	GetExchangeAssetIcon(icon string) (string, string, error)
 	GetExchangeFileContent(link, packaging, mainFile string) ([]byte, error)
-	GetAnalyticsWindow() ([]AnalyticsEvent, error)
```

Comment:
The `Client` interface should still have the GetAnalyticsWindow method on here.
```
GetAnalyticsWindow(string, string) ([]AnalyticsEvent, error)
```

Reply:
Is Client a superset? I guess I did not see the point- I will update